### PR TITLE
Fix/format

### PR DESCRIPTION
**Pull request by RedJue**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [x] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |   Repair code format as expected      |
| 🇨🇳 Chinese |     按照预期修复代码格式      |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -45,7 +45,9 @@
     "prepublishOnly": "npm run compile && np --no-cleanup --yolo --no-publish",
     "lint": "eslint src/ --ext .tsx,.ts",
     "lint:tsc": "tsc -p tsconfig.json --noEmit",
-    "now-build": "npm run docs:build"
+    "now-build": "npm run docs:build",
+    "husky-hook": "husky add .husky/pre-commit 'npx --no-install lint-staged'",
```
Comment (MadCcc) [inline package.json:49]: 这条可以不用

Comment (RedJue) [reply package.json:49]: 好 那我去了

**Comment by MadCcc** [comment]
ci 的 node 改成 14 试试

**Comment by MadCcc** [comment]
npm 7 会在 `npm i --package-lock-only` 后执行生命周期脚本，是个 bug
https://github.com/npm/cli/issues/2787

**Comment by MadCcc** [comment]
或者改成 `npm i --package-lock-only --ignore-scripts`

**Pull request merged**